ts/src/apex.ts

```diff
@@ -0,0 +1,1940 @@
+     * @description fetches information on multiple orders made by the user
+     * @see https://api-docs.pro.apex.exchange/#privateapi-v3-for-omni-get-open-orders
+     * @param {string} symbol unified market symbol of the market orders were made in
+     * @param {int} [since] the earliest time in ms to fetch orders for
+     * @param {int} [limit] the maximum number of order structures to retrieve
+     * @param {object} [params] extra parameters specific to the exchange API endpoint
+     * @returns {Order[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+     */
+    async fetchOpenOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Order[]> {
+        const response = await this.privateGetV3OpenOrders ();
+        const orders = this.safeList (response, 'data', []);
+        return this.parseOrders (orders, undefined, since, limit);
+    }
+
+    /**
+     * @method
+     * @name apex#fetchOrders
+     * @description fetches information on multiple orders made by the user *classic accounts only*
+     * @see https://api-docs.pro.apex.exchange/#privateapi-v3-for-omni-post-cancel-all-open-orders
+     * @param {string} symbol unified market symbol of the market orders were made in
+     * @param {int} [since] the earliest time in ms to fetch orders for
+     * @param {int} [limit] the maximum number of order structures to retrieve, default 100
+     * @param {object} [params] extra parameters specific to the exchange API endpoint
+     * @param {object} [params.endTimeExclusive] end time
+     * @param {boolean} [params.status] "PENDING", "OPEN", "FILLED", "CANCELED", "EXPIRED", "UNTRIGGERED"
+     * @param {boolean} [params.side] BUY or SELL
+     * @param {string} [params.type] "LIMIT", "MARKET","STOP_LIMIT", "STOP_MARKET", "TAKE_PROFIT_LIMIT","TAKE_PROFIT_MARKET"
+     * @param {string} [params.orderType] "ACTIVE","CONDITION","HISTORY"
+     * @param {boolean} [params.page] Page numbers start from 0
+     * @returns {Order[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+     */
+    async fetchOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Order[]> {
+        await this.loadMarkets ();
+        const request: Dict = {};
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['symbol'] = market['id'];
+        }
+        if (since !== undefined) {
+            request['beginTimeInclusive'] = since;
+        }
+        const status = this.safeString (params, 'status');
+        if (status !== undefined) {
+            request['status'] = status;
+        }
+        const type = this.safeString (params, 'type');
+        if (type !== undefined) {
+            request['type'] = type;
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
```

Comment:
reading from `params` just to add it to `request` is not needed, because at the end we should extend request with params